grid_map:
  # Grid_map parameters
  resolution: 0.05
  local_update_range_x: 10
  local_update_range_y: 10
  local_update_range_z: 4.5
  obstacles_inflation: 0.0
  local_map_margin: 10
  ground_height: -1.0
  # Intrinsic parameters of the depth camera
  cx: 321.04638671875
  cy: 243.44969177246094
  fx: 387.229248046875
  fy: 387.229248046875
  # Depth filter
  use_depth_filter: true
  depth_filter_tolerance: 0.15
  depth_filter_maxdist: 5.0
  depth_filter_mindist: 0.2
  depth_filter_margin: 2
  k_depth_scaling_factor: 1000.0
  skip_pixel: 2
  # Local fusion
  p_hit: 0.65
  p_miss: 0.35
  p_min: 0.12
  p_max: 0.90
  p_occ: 0.80
  min_ray_length: 0.1
  max_ray_length: 4.5
  visualization_truncate_height: 2.8
  virtual_ceil_height: 3.0
  show_occ_time: false
  frame_id: "world"
  # SDF map
  local_bound_inflate: 0.0
  show_esdf_time: false
  esdf_slice_height: 1.6